app/scripts/controllers/pod.js

```diff
@@ -128,17 +128,19 @@ angular.module('openshiftConsole')
       });
     };
 
-    if (!characterBoundingBox.height || !characterBoundingBox.width) {
-      Logger.warn("Unable to calculate the bounding box for a character.  Terminal will not be able to resize.");
-    }
-    else {
-      $(window).on('resize.terminalsize', _.debounce(calculateTerminalSize, 100));
-    }
-
     $scope.$watch('selectedTab.terminal', function(terminalTabSelected) {
       if (!!terminalTabSelected) {
+        if (!characterBoundingBox.height || !characterBoundingBox.width) {
+          Logger.warn("Unable to calculate the bounding box for a character.  Terminal will not be able to resize.");
+        }
+        else {
+          $(window).on('resize.terminalsize', _.debounce(calculateTerminalSize, 100));
+        }        
         $timeout(calculateTerminalSize, 0);
       }
+      else {
+        $(window).off('resize.terminalsize');
```

Comment:
Do we still need the `off()` call in destroy? What if I navigate away with the terminal tab selected?